recipe/emailverification/evmodels/models.go

```diff
@@ -20,19 +20,29 @@ import (
 	"github.com/supertokens/supertokens-golang/supertokens"
 )
 
+type TypeGetEmailForUserID func(userID string, userContext supertokens.UserContext) (TypeEmailInfo, error)
+
+type TypeEmailInfo struct {
+	OK *struct {
+		Email string
+	}
+	EmailDoesNotExistError *struct{}
+	UnknownUserIDError     *struct{}
+}
+
 type TypeInput struct {
-	GetEmailForUserID        func(userID string, userContext supertokens.UserContext) (string, error)
-	GetEmailVerificationURL  func(user User, userContext supertokens.UserContext) (string, error)
+	Mode                     string
```

Comment:
Make this an enum

Reply:
Done

Reply:
How does this work? It still seems to be a string only cause somewhere you are doing `if config.Mode == "REQUIRED" {`. So if a user gives another value, what will happen?